layouts/partials/comments/isso.html

```diff
@@ -0,0 +1,6 @@
+{{- $issoURL := .Page.Site.Params.comments.isso.url -}}
+
+<script data-isso="{{ $issoURL }}"
```

Comment:
Is `data-isso` a required option?

Reply:
According to the docs, no: 
> Isso usually detects the REST API automatically, but when you serve the JS script on a different location, this may fail. Use data-isso to override the API location

...but every single article or blog post about isso I've read includes both `data-isso` and `src` in their integration code, including the [Isso quickstart guide](https://posativ.org/isso/docs/quickstart/#integration). It really is the recommended way.

Reply:
How about, we simplify the config block to this:

```toml
[params.comments.isso]
enable = false
scriptSrc = "" # "https://isso.example.com/js/embed.min.js"
dataAttrs = "" # "data-isso='https://isso.example.com'"
```

- This explicitly points to the script src
- Doesn't have to deal with if `url` has trailing slash or not
- `data-isso` and other `data-` attributes go together under single option

Reply:
Yeah, why not. It may be a little simpler. I'll push the updated commit soon. Thanks for your review!